crates/hir-ty/src/lower.rs

```diff
@@ -746,6 +717,8 @@ impl<'a> TyLoweringContext<'a> {
             res,
             Some(segment.name.clone()),
             move |name, t, associated_ty| {
+                let generics = self.generics().unwrap();
```

Comment:
This seems unnecessary, we already have the generics in scope from line 740/711 above (or is that due to borrowing issues?)

Reply:
Ah nvm it likely is